README.md

```diff
@@ -32,8 +33,7 @@ This process will open browser windows to interact with the DevCycle universal l
 To switch organizations once logged in, the [org command](docs/org.md) can be used.
 ## Using Client Credentials
 ### Credentials File
-Create a subdirectory inside the directory where you're running the CLI called `.devcycle`, then inside that directory
-create an `auth.yml` file with the following contents:
+Create a `.devcycle` subdirectory inside your home directory. Inside that directory create an `auth.yml` file with the following contents:
```

Comment:
is this correct? I think by default it uses `.config/oex`, we should also document the repo config

Reply:
You're right, but I believe this is platform-specific. I should link out to the oclif readme section that covers default config paths.

I'll also add an output line whenever the config is saved to say where it was saved.

Reply:
Ah, discovered a configuration issue. "oex" should be an application-specific folder name. Changing that to `devcycle`